cloud/blockstore/libs/nbd/netlink_device.cpp

```diff
@@ -108,9 +108,9 @@ TFuture<NProto::TError> TNetlinkDevice::Start()
         ConnectSocket();
         Connect();
 
-    } catch (const TServiceError& e) {
+    } catch (const std::exception& e) {
```

Comment:
The reasoning behind handling only TServiceError was that we wanted the caller to deal with ConnectSocket exceptions, but in reality we forgot to handle it at all and just crashed the proxy